src/Microsoft.TemplateSearch.TemplateDiscovery/PackChecking/PackSourceChecker.cs

```diff
@@ -73,7 +73,8 @@ await foreach (IPackInfo sourceInfo in packProvider.GetCandidatePacksAsync().Con
                             Verbose.WriteLine("Found templates:");
                             foreach (ITemplateInfo template in packCheckResult.FoundTemplates)
                             {
-                                Verbose.WriteLine($"  - {template.Identity} ({template.ShortName}) by {template.Author}, group: {(string.IsNullOrWhiteSpace(template.GroupIdentity) ? "<not set>" : template.GroupIdentity)}, precedence: {template.Precedence}");
+                                string shortNames = string.Join(",", template.ShortNameList);
```

Comment:
Should we use `", "`(with space after `,`) in this scenarios?

Reply:
we discussed it some time ago, for displaying languages and agreed not to use space after comma to save space in the tables.
Short name is single word so this should be readable. I hope no one puts spaces and punctuation to short name :)